src/js/components/stock-movement-wizard/CreateStockMovement.jsx

```diff
@@ -288,11 +328,21 @@ class CreateStockMovement extends Component {
   }
 }
 
-export default withRouter(connect(null, {
+const mapStateToProps = state => ({
+  locationName: state.location.currentLocation.name,
```

Comment:
why this is separated into 3 different props, I think it will be simpler to have only one prop named location or currentLocation and assign the currentLocation to it